app/src/components/profile/public/ProfileHeaderLinks.tsx

```diff
@@ -30,7 +39,14 @@ export default function ProfileHeaderLinks({
         <BubbleLink
           href={`https://github.com/${user.github}`}
           icon="/assets/icons/github-icon.svg"
-          text={<span className="text-sm text-black">@{user.github}</span>}
+          text={
+            <>
+              <span className="text-sm text-black">@{user.github}</span>
+              <span className="text-sm text-[#404454]">
```

Comment:
```suggestion
              <span className="text-sm text-gray-500">
```

let's use predefined colors. It's likely that they're not matching Firgma 1:1 tho, that's okay by now. At some point, we'll need to redefine the global config